src/frontend/components/LoginBanner.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import LoginBanner from './LoginBanner'
+
+
+describe('<LoginBanner/> rendering', ()=>{
+    it('renders correctly', ()=>{
+        const bannerComponent = renderer
+        .create(<LoginBanner/>)
```

Comment:
Consistency in code indentation is very key. Please consider using Prettier in VS code